regression/esbmc/github_666/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+no_slice.c
+
+^VERIFICATION FAILED$
```

Comment:
Since this focus on no-slice, could you add some regex to show that the symbol is shown at the counter example?

Reply:
Good catch, done. Would be a bit useless without.